test/conventions.jl

```diff
@@ -0,0 +1,42 @@
+@testset "Conventions" begin
```

Comment:
That is awesome, thanks for the additional tests! ❤️ Could you move them to the test/neighborhood.jl file together with the other tests of similar nature? We have an open issue regarding these conventions that may justify a separate test/conventions.jl file in the future: #43